raphtory/src/db/api/mutation/import_ops.rs

```diff
@@ -50,6 +54,33 @@ pub trait ImportOps:
         force: bool,
     ) -> Result<NodeView<Self, Self>, GraphError>;
 
+    /// Imports a single node into the graph.
+    ///
+    /// This function takes a reference to a node and an optional boolean flag `force`.
+    /// If `force` is `Some(false)` or `None`, the function will return an error if the node already exists in the graph.
```

Comment:
Seeing as you have to give a value should we change this to a bool instead of Option Bool